custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt

```diff
@@ -62,6 +62,26 @@ class CloudIssuePage(
         return CloudIssueLinking(driver)
     }
 
+    override fun changeAssignee(): CloudIssuePage {
+        driver
+            .findElement(By.cssSelector("[data-test-id = 'issue.views.field.user.assignee']"))
+            .click();
+
+        var firstUser = driver
```

Comment:
Let's use `val`s by default. Immutable values are much easier to work with than mutable variables.

Reply:
HTML for popup element looks like. What immutable values would you recommend to take?

<div class="css-dukhv4-menu fabric-user-picker__menu"><div class="css-1ljkvdv fabric-user-picker__menu-list"><div class="css-cp87li-option fabric-user-picker__option fabric-user-picker__option--is-focused" id="react-select-assignee-option-0" tabindex="-1"><div class="styledCache__StyledCustomComponent-zohhd2-0 hMSbZe sc-eTyWNx hjmYYA"><div style="display: inline-block; position: relative; outline: 0px; height: 36px; width: 36px;"><span class="styledCache__StyledSpan-zohhd2-3 hDJQZm"><svg viewBox="0 0 128 128" version="1.1" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="Unassigned" style="background-color: transparent; border-radius: 50%; height: 100%; width: 100%;"><g class="AvatarImage__ShapeGroup-sc-1hf6c4m-0 lcCoJI" mode="light"><circle cx="64" cy="64" r="64"></circle><g><path d="M103,102.1388 C93.094,111.92 79.3504,118 64.1638,118 C48.8056,118 34.9294,111.768 25,101.7892 L25,95.2 C25,86.8096 31.981,80 40.6,80 L87.4,80 C96.019,80 103,86.8096 103,95.2 L103,102.1388 Z"></path><path d="M63.9961647,24 C51.2938136,24 41,34.2938136 41,46.9961647 C41,59.7061864 51.2938136,70 63.9961647,70 C76.6985159,70 87,59.7061864 87,46.9961647 C87,34.2938136 76.6985159,24 63.9961647,24"></path></g></g></svg></span></div><div class="AvatarItem__Content-sc-189r3pd-0 gBVfpU"><div class="AvatarItem__PrimaryText-sc-189r3pd-1 eMREul"><span class="sc-ccXozh kpIETV" color="#172B4D">Unassigned</span></div><div class="AvatarItem__SecondaryText-sc-189r3pd-2 EKGYc"></div></div></div></div></div></div>

Reply:
It's about Kotlin, not HTML
```suggestion
        val firstUser = driver
```